api/server.go

```diff
@@ -206,6 +222,16 @@ func GetHttpServer(meter RelayMeter, l *logger.Logger) func(w http.ResponseWrite
 			return
 		}
 
+		if appID := match(appsLatencyPath, req.URL.Path); appID != "" {
```

Comment:
These need to happen before the `totalRelays` match: otherwise they may not be reached.

Reply:
Moved above the `totalRelays` match